src/components/ProjectSettingsTabs/GithubCommitQueueTab/getFormSchema.ts

```diff
@@ -194,6 +194,14 @@ export const getFormSchema = (
               repoData?.commitQueue?.enabled
             ),
           },
+          requireSigned: {
```

Comment:
I think this should be hidden if the commit queue isn't enabled, since it doesn't do anything.